src/tsickle.ts

```diff
@@ -659,7 +660,11 @@ class Annotator extends ClosureRewriter {
         // Only emit a type annotation when it's a plain variable and
         // not a binding pattern, as Closure doesn't(?) have a syntax
         // for annotating binding patterns.  See issue #128.
-        if (varDecl.name.kind === ts.SyntaxKind.Identifier) {
+        // Don't emit type annotation when the variable statement is a @polymerBehavior,
+        // as otherweise the polymer closure checker will fail.
+        // TODO(tbosch): file an issue for this!
```

Comment:
Created https://github.com/angular/tsickle/issues/529 to track this and linked in the code.